docs/source/models/gbd2017/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -23,44 +23,58 @@ Scope of Modeling
 Concept Model Diagram
 ---------------------
 
-The following four diagrams are placeholders; they will be combined into a
-single model encompassing all three interventions.
+.. image:: lsff_concept_model_diagram.svg
 
-Vitamin A Fortification
-+++++++++++++++++++++++
+Model Components
+----------------
 
-.. image:: vitamin_a_diagram.svg
+Demographics
+++++++++++++
 
-Iron Fortification
-++++++++++++++++++
+GBD Causes
+++++++++++
 
-.. image:: iron_neonatal_diagram.svg
+:ref:`Measles <2017_cause_measles>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: iron_detail_diagram.svg
+:ref:`Diarrheal Diseases <2017_cause_diarrhea>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Folic Acid Fortification
-++++++++++++++++++++++++
+:ref:`Neural Tube Defects <2017_cause_neural_tube_defects>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: folic_acid_diagram.svg
+Lower Respiratory Infections
```

Comment:
Can you go ahead and make stubs for all these other cause docs?

Reply:
Yes, I'll create stubs for the missing ones. Lu created the LRI document, so that should be merged into master now.

Reply:
@collijk Three questions:

1. Do we want to keep adding cause model documents in files directly in the gbd2017/causes folder, or would it be better to create a subfolder for each cause that includes the main page (renamed to index.rst to allow easy globbing?) as well as the image files, etc.?

2. Would it be better to push the requested changes to this pull request, then merge, or merge first, then create a new pull request adding stubs for missing causes and renaming the "Risk-Cause Relationships" section to "Risk-Outcome Relationships"?

3. Where should I put the stubs for the PAF-of-1 causes?  In the cause folder, or in a separate folder for "risk-attributable causes"?